docs/developer/transform.rst

```diff
@@ -70,11 +75,34 @@ Note that the template argument ``std::tuple<bool>`` is equivalent to ``std::tup
 Recommended usage
 -----------------
 
-For improved testability and maintainability it is recommended to define the operator stand-alone instead of inline.
-The list of supported types can also be provided in this manner.
+- For improved testability and maintainability it is recommended to define the operator stand-alone instead of inline.
+- Specifying types on the element wise operator allows them to be omitted from 
+``transform`` and ``transform_in_place``.
+- Use ``arg_list`` to define list of supported type combinations as shown below.
+
+.. code-block:: cpp
+
+  // This will accept a pair of (double, bool) / (float, int) / ...etc.
+  overloaded{arg_list<std::tuple<double, bool>,
+                      std::tuple<float, int>,
```

Comment:
```suggestion
  // This will accept a pair of (double, bool) / (float, int64_t) / ...etc.
  overloaded{arg_list<std::tuple<double, bool>,
                      std::tuple<float, int64_t>,
```